apps/modernization-ui/src/apps/page-builder/components/Concept/Concept.tsx

```diff
@@ -32,32 +74,84 @@ export const Concept = () => {
     const [isShowFrom, setShowForm] = useState(false);
     const [isDelete, setIsDelete] = useState(false);
     const [concept, setConcept] = useState(initConcept);
+    const [codeSystemOptionList, setCodeSystemOptionList] = useState<CodeSystemOption[]>([]);
 
+    const handleValidation = (unique = true) => {
+        return unique ? /^[a-zA-Z0-9_]*$/ : /^[a-zA-Z0-9\s?!,-_]*$/;
+    };
     // @ts-ignore
```

Comment:
Issue: I know you didn't add this initially, but can you please remove the `@ts-ignore`